physionet-django/user/migrations/0051_remove_user_associated_emails_allowed.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.1.14 on 2022-12-16 21:00
```

Comment:
This migration can be removed

Reply:
Thank you. Removing them. i was also wondering the same and asked Harry if that would be okay